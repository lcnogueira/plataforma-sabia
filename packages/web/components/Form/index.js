export { default as InputField } from './InputField';
export { default as SwitchField } from './SwitchField';
export { default as TextField } from './TextField';
export { default as SelectField } from './SelectField';
export { default as CheckBoxField } from './CheckBoxField';
export { default as MaskedInputField } from './MaskedInputField';
export { default as CurrencyInputField } from './CurrencyInputField';
export { default as Watcher } from './Watcher';
export { default as InputHiddenField } from './InputHiddenField';
export { default as Repeater } from './Repeater';
export { default as Help } from './Help';
export { default as HelpModal } from './HelpModal';
export * from './Form';
