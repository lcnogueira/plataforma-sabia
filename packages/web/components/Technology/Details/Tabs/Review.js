import React, { useCallback, useState } from 'react';
import { useTechnology } from '../../../../hooks';
import { getReviews } from '../../../../services/technology';
import * as Layout from '../../../Common/Layout';
import Loading from '../../../Loading';
import Section from '../Section';

const Review = () => {
	const { technology } = useTechnology();

	const [reviews, setReviews] = useState(technology.reviews);
	const [loading, setLoading] = useState(false);
	const [orderBy, setOrderBy] = useState({
		orderBy: 'created_at',
		order: 'DESC',
	});

	const updateReviewsData = useCallback(async () => {
		setLoading(true);

		const response = await getReviews(technology.id, orderBy);
		setReviews(response);

		setLoading(false);
	}, [orderBy, technology.id]);

	const reviewSelectOptions = [
		{ label: 'Mais recentes', value: 'created_at|DESC' },
		{ label: 'Mais Bem Avaliados', value: 'rating|DESC' },
		{ label: 'Mais Antigos', value: 'created_at|ASC' },
	];

	const updateReviewSelectOrder = (selected) => {
		const [selectedOrderBy, selectedOrder] = selected.split('|');
		setOrderBy({ orderBy: selectedOrderBy, order: selectedOrder });
		updateReviewsData();
	};

	return (
		<Layout.Cell>
			<Section title="Relatos" hideWhenIsEmpty={false}>
				<select
					name="reviews"
					onChange={(event) => {
						updateReviewSelectOrder(event.target.value);
					}}
				>
					{reviewSelectOptions.map((option) => (
						<option value={option.value}>{option.label}</option>
					))}
				</select>

				{loading ? (
					<Loading />
				) : (
					!!reviews &&
					reviews?.map((review) => (
						<>
							<p>{review.id}</p>
						</>
					))
				)}
			</Section>
		</Layout.Cell>
	);
};

export default Review;
