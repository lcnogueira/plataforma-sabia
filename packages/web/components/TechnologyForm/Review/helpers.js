export const fundingData = {
	types: [
		{
			value: 'public',
			label: 'Público',
		},
		{
			value: 'private',
			label: 'Privado',
		},
		{
			value: 'collective',
			label: 'Coletivo',
		},
	],
	status: [
		{
			value: 'not_acquired',
			label: 'Não adquirido',
		},
		{
			value: 'acquiring',
			label: 'Em aquisição',
		},
		{
			value: 'acquired',
			label: 'Já adquirido',
		},
	],
};

/**
 * Returns funding label based on value.
 *
 * @param {string} scope The scope of funding.
 * @param {string} value The funding value
 * @returns {string} The funding label.
 */
export const getFundingLabelByValue = (scope, value) => {
	const keys = Object.keys(fundingData);

	if (!scope || !keys.some((key) => key === scope)) {
		return value;
	}

	const funding = fundingData[scope].find((data) => data.value === value);

	return funding?.label || value;
};
