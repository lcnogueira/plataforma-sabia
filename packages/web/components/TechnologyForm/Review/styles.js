import styled from 'styled-components';
import * as Layout from '../../Common/Layout';

export const Cell = styled(Layout.Cell)`
	width: 100%;
`;

export const Row = styled.div`
	display: flex;
	align-items: flex-start;

	& > *:not(:first-child):not(:last-child) {
		margin: 0 10px;
	}

	@media (max-width: ${({ theme }) => theme.screens.large}px) {
		flex-direction: column;
		margin-top: 1.5rem;
	}
`;

export const Col = styled.div`
	flex: ${({ size }) => size || 1};

	@media (max-width: ${({ theme }) => theme.screens.small}px) {
		width: 100%;
	}
`;

export const Wrapper = styled.div`
	margin-bottom: 4rem;
`;

export const Label = styled.label`
	display: block;
	margin: 1rem 0;
	text-transform: uppercase;

	a {
		padding: 0;
	}

	input[type='checkbox'] {
		margin-right: 1rem;
	}
`;

export const Checkbox = styled.input.attrs(({ name }) => ({
	name,
	id: name,
	type: 'checkbox',
}))`
	background: blue;
	border: 1px solid red;
`;
