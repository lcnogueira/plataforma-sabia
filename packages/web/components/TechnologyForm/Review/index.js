import React from 'react';
import PropTypes from 'prop-types';
import * as Layout from '../../Common/Layout';
import Repeater from '../../Form/Repeater';
import Section from '../../Technology/Details/Section';
import TextValue from '../../Technology/Details/TextValue';
import { Row, Wrapper } from './styles';

const Review = ({ form }) => {
	const technology = form.getValues();

	return (
		<Wrapper>
			<Repeater
				form={form}
				name="review"
				childsComponent={({ item }) => (
					<Row key={item.id}>
						<Layout.Cell col="2">
							<Section
								title="Identificação"
								color="lightGray"
								hideWhenIsEmpty={false}
							>
								<TextValue title="Título" value={technology.title} />
								<TextValue
									title="Nome Popular"
									value={technology.taxonomies?.popular_name}
								/>
								<TextValue title="Sigla" value={technology.taxonomies?.initials} />
								<TextValue title="Descrição" value={technology.description} />
								<TextValue
									title="Categoria"
									value={technology.taxonomies?.category}
								/>
								<TextValue
									title="Classificação"
									value={technology.taxonomies?.classification}
								/>
								<TextValue
									title="Dimensão"
									value={technology.taxonomies?.dimension}
								/>
								<TextValue
									title="Público-alvo"
									value={technology.taxonomies?.target_audience}
								/>
								<TextValue title="Bioma" value={technology.taxonomies?.biome} />
								<TextValue
									title="Programa Governamental"
									value={technology.taxonomies?.government_program}
								/>
								<TextValue
									title="Palavras-chave"
									value={technology.taxonomies?.keywords}
								/>
							</Section>

							<Section
								title="Aspectos Legais"
								color="lightGray"
								hideWhenIsEmpty={false}
							>
								<TextValue
									title="Tecnologia Patenteada"
									value={technology.patent ? 'Sim' : 'Não'}
								/>
								<TextValue
									title="Proteção Intelectual"
									value={technology.taxonomies?.intellectual_property}
								/>
							</Section>

							<Section
								title="Estágio de Desenvolvimento"
								color="lightGray"
								hideWhenIsEmpty={false}
							>
								<TextValue
									title="Escala TRL"
									value={technology.taxonomies?.stage}
								/>
							</Section>

							<Section
								title="Financiamento"
								color="lightGray"
								hideWhenIsEmpty={false}
							>
								<TextValue
									title="Necessita de Financiamento"
									value={technology.technologyCosts?.costs?.funding_required}
								/>
								<TextValue
									title="Tipo de Financiamento"
									value={technology.technologyCosts?.costs?.funding_type}
								/>
								<TextValue
									title="Valor do Financiamento"
									value={technology.technologyCosts?.costs?.funding_value}
								/>
								<TextValue
									title="Situação"
									value={technology.technologyCosts?.costs?.funding_status}
								/>
							</Section>
						</Layout.Cell>

						<Layout.Cell col="2">
							<Section title="Objetivos" color="lightGray" hideWhenIsEmpty={false}>
								<TextValue
									title="Objetivo Principal"
									value={technology.taxonomies?.primary_objective}
								/>
								<TextValue
									title="Objetivos secundários"
									value={technology.taxonomies?.secondary_objectives}
								/>
							</Section>

							<Section
								title="Problematização"
								color="lightGray"
								hideWhenIsEmpty={false}
							>
								<TextValue
									title="Problemas que a tecnologia soluciona"
									value={technology.taxonomies?.problematization}
								/>
								<TextValue
									title="Problemas que a tecnologia acarreta"
									value={technology.taxonomies?.problems_caused}
								/>
							</Section>

							<Section title="Aplicação" color="lightGray" hideWhenIsEmpty={false}>
								<TextValue
									title="Onde é a Aplicação"
									value={technology.taxonomies?.locale}
								/>
								<TextValue
									title="Aplicação"
									value={technology.taxonomies?.application}
								/>
								<TextValue
									title="Exemplos de Aplicação"
									value={technology.taxonomies?.application_examples}
								/>
								<TextValue
									title="Pré-requisitos para a implantação"
									value={technology.taxonomies?.prerequisites_for_deployment}
								/>
								<TextValue
									title="Duração do processo de instalação da tecnologia"
									value={technology.taxonomies?.installation_time}
								/>
							</Section>

							<Section title="Contribuição" color="lightGray" hideWhenIsEmpty={false}>
								<TextValue
									title="Contribuição para o semiárido"
									value={technology.taxonomies?.contribution_to_semiarid}
								/>
							</Section>

							<Section title="Riscos" color="lightGray" hideWhenIsEmpty={false}>
								<TextValue
									title="Riscos associados à tecnologia"
									value={technology.taxonomies?.riskiness}
								/>
							</Section>
						</Layout.Cell>
					</Row>
				)}
			/>
		</Wrapper>
	);
};

Review.propTypes = {
	form: PropTypes.shape({
		getValues: PropTypes.func.isRequired,
	}),
};

Review.defaultProps = {
	form: {},
};

export default Review;
