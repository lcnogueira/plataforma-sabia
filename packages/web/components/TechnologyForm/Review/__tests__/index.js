import React from 'react';
import { render } from 'test-utils';
import { getFakeTechnology, normalizeAttachments } from '@sabia/core';
import Review from '..';
import { Form } from '../../../Form';

let technology = getFakeTechnology();

technology = {
	...technology,
	attachments: normalizeAttachments(technology.attachments),
};

const data = {
	technology,
};

const onSubmit = jest.fn(() => {});

test('it render the review page', () => {
	const { container } = render(
		<Form onSubmit={onSubmit}>
			<Review data={data} />
		</Form>,
	);

	expect(container).toMatchSnapshot();
});
