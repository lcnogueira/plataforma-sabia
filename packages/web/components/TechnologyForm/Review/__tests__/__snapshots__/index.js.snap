// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it render the review page 1`] = `
.c23 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 1rem;
  font: 1.2rem;
}

.c24 {
  width: 2.5rem;
  height: 2.5rem;
  margin: 0rem;
  margin: 0 0.5rem 0 0;
}

.c25 {
  width: 100%;
  color: hsla(0,0%,40%);
}

.c0 {
  margin: 1rem auto;
  padding: 2rem 1rem;
  width: 100%;
}

.c0 button,
.c0 a {
  padding-right: 3rem;
  padding-left: 3rem;
}

.c0 > label {
  width: 100%;
  display: block;
  font-weight: 700;
  font-size: 1.7rem;
}

.c4 {
  margin-left: 1rem;
  padding: 2rem 0;
}

.c5 {
  border-bottom: 4px solid hsla(0,0%,40%);
  width: calc(100% - 1rem);
  margin: 2rem 0;
}

.c5:first-child {
  margin-top: 0;
}

.c5 h4 {
  display: inline-block;
  background-color: hsla(0,0%,40%);
  color: hsla(0,0%,100%);
  padding: 2.5rem 3rem;
  font-weight: 600;
  font-size: 1.8rem;
  line-height: 2rem;
  text-transform: uppercase;
  margin-bottom: -4px;
}

.c6 {
  font-size: 1.6rem;
  line-height: 2.4rem;
}

.c6 strong {
  color: hsla(0,0%,7%);
}

.c6 span {
  color: hsla(204,7%,15%);
}

.c7 {
  width: 100%;
  height: 100%;
  min-height: 20rem;
  background: hsla(0,0%,100%);
  border: 0.1rem solid hsla(0,0%,80%);
  border-radius: 0.2rem;
  margin-top: 3rem;
}

.c7 section {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 0.8rem;
  width: 100%;
  height: 100%;
  overflow: auto;
}

.c8 {
  font-size: 1.6rem;
  font-style: normal;
  font-weight: bold;
  background: hsla(0,0%,95%);
  width: 100%;
  padding: 1.2rem;
  text-transform: uppercase;
  color: hsla(204,7%,15%);
  border-bottom: 0.1rem solid hsl(0,0%,74%);
}

.c9 {
  border-collapse: collapse;
  width: 100%;
  color: hsla(204,7%,15%);
  overflow: scroll;
}

.c9 th,
.c9 td {
  padding: 0.8rem 0.5rem;
  border-bottom: 0.1rem solid hsl(0,0%,74%);
}

.c9 th:not(:first-child),
.c9 td:not(:first-child) {
  text-align: center;
}

.c9 tr:first-of-type {
  border-bottom: 0.2rem solid hsl(0,0%,74%);
}

.c9 tr:nth-child(even) {
  background-color: hsla(0,0%,95%);
}

.c9 tr th {
  font-weight: 500;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  font-style: normal;
  font-weight: bold;
  font-size: 1.6rem;
  line-height: 2rem;
  padding: 2rem;
  color: hsla(146,64%,36%);
}

.c11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  font-style: normal;
  font-weight: bold;
  font-size: 1.6rem;
  line-height: 2rem;
  padding: 2rem;
  color: hsla(204,70%,53%);
}

.c12 {
  color: hsla(204,7%,15%);
  text-align: center;
}

.c20 {
  width: 100%;
  height: 100%;
  background: hsla(0,0%,100%);
  margin-top: 3rem;
  background: transparent;
}

.c20 section {
  padding: 0.8rem;
  width: 100%;
  height: 100%;
  background: transparent;
  overflow-x: auto;
  overflow-y: hidden;
}

.c21 {
  border-collapse: collapse;
  width: 100%;
  color: hsla(204,7%,15%);
}

.c21 th,
.c21 td {
  padding: 0.8rem 0.5rem;
  border-bottom: 0.1rem solid hsl(0,0%,74%);
}

.c21 th:not(:first-child),
.c21 td:not(:first-child) {
  text-align: center;
}

.c21 thead th {
  font-weight: 500;
  border-bottom: 0.2rem solid hsl(0,0%,74%);
}

.c21 tbody tr:nth-child(odd) {
  background-color: hsl(0,0%,93%);
}

.c3 {
  -webkit-flex: 2;
  -ms-flex: 2;
  flex: 2;
  margin: 0 1rem 0 0;
  text-align: left;
  width: 100%;
}

.c22 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  margin: 0 1rem 0 0;
  text-align: left;
  width: 100%;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
}

.c2 > *:not(:first-child):not(:last-child) {
  margin: 0 10px;
}

.c1 {
  margin-bottom: 4rem;
}

.c13 {
  display: block;
  font-weight: 500;
  font-size: 1.4rem;
  margin-bottom: 1rem;
  text-transform: uppercase;
  color: hsla(0,0%,40%);
}

.c13:not(:first-child) {
  margin-top: 1rem;
}

.c17 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c14 {
  display: grid;
  grid-template-columns: repeat(auto-fill,30%);
  grid-gap: 1rem;
}

.c15 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c15 > * {
  margin-bottom: 1rem;
}

.c15 > a {
  padding: 0;
  margin-right: 1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c18 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c18 > * {
  margin-bottom: 1rem;
}

.c18 > a {
  padding: 0;
  margin-right: 1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c16 {
  width: 100%;
  -webkit-filter: opacity(0.7);
  filter: opacity(0.7);
}

.c19 {
  color: hsla(0,0%,7%);
}

@media (max-width:450px) {
  .c7 {
    max-width: 40rem;
  }
}

@media (max-width:1170px) {
  .c20 {
    max-width: 80rem;
  }
}

@media (max-width:768px) {
  .c20 {
    max-width: 65rem;
    margin: auto;
  }
}

@media (max-width:1170px) {
  .c3 {
    max-width: initial;
    margin: 0 0 1rem 0;
  }
}

@media (max-width:768px) {
  .c3 {
    margin: 0;
  }
}

@media (max-width:1170px) {
  .c22 {
    max-width: initial;
    margin: 0 0 1rem 0;
  }
}

@media (max-width:768px) {
  .c22 {
    margin: 0;
  }
}

@media (max-width:1170px) {
  .c2 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    margin-top: 1.5rem;
  }
}

@media (max-width:450px) {
  .c14 {
    grid-template-columns: repeat(auto-fill,100%);
  }
}

<div>
  <form
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="Layout__Cell-m6jxy6-4 c3"
        >
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Identificação
              </h4>
            </div>
            <p
              class="c6"
            >
              <strong>
                Título
                : 
              </strong>
              <span>
                Gesbib powev sodzomjikm.
              </span>
            </p>
            <p
              class="c6"
            >
              <strong>
                Descrição
                : 
              </strong>
              <span>
                Kodopifi sammehor voecara wu dietu je zedsande lo nemnoriv zi metoeju na bajo.
              </span>
            </p>
            <p
              class="c6"
            >
              <strong>
                Classificação
                : 
              </strong>
              <span>
                Inovações sociais
              </span>
            </p>
            <p
              class="c6"
            >
              <strong>
                Palavras-chave
                : 
              </strong>
              <span>
                Mouse, Ladybug, Amur Tiger, Rabbit, Carp
              </span>
            </p>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Aspectos Legais
              </h4>
            </div>
            <p
              class="c6"
            >
              <strong>
                Tecnologia Patenteada
                : 
              </strong>
              <span>
                common:yes
              </span>
            </p>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Estágio de Desenvolvimento
              </h4>
            </div>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Financiamento
              </h4>
            </div>
            <p
              class="c6"
            >
              <strong>
                Necessita de Financiamento
                : 
              </strong>
              <span>
                common:yes
              </span>
            </p>
            <p
              class="c6"
            >
              <strong>
                Tipo de Financiamento
                : 
              </strong>
              <span>
                Privado
              </span>
            </p>
            <p
              class="c6"
            >
              <strong>
                Valor do Financiamento
                : 
              </strong>
              <span>
                92549056
              </span>
            </p>
            <p
              class="c6"
            >
              <strong>
                Situação
                : 
              </strong>
              <span>
                Já adquirido
              </span>
            </p>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Custos da Tecnologia
              </h4>
            </div>
            <div
              class="c7"
            >
              <div
                class="c8"
              >
                Custo de Desenvolvimento
              </div>
              <section>
                <table
                  class="c9"
                >
                  <thead>
                    <tr>
                      <th>
                        Descricao
                      </th>
                      <th>
                        Tipo
                      </th>
                      <th>
                        Qtde
                      </th>
                      <th>
                        Valor
                      </th>
                      <th>
                        Total
                      </th>
                    </tr>
                  </thead>
                  <tbody>
                    <tr>
                      <td>
                        Rojas uzvajka pi padtiz wozcum hidudasu ovwebgoh peab ju liugcod.
                      </td>
                      <td>
                        Insumo
                      </td>
                      <td>
                        47
                      </td>
                      <td>
                        R$84,262,938.00
                      </td>
                      <td>
                        R$3,960,358,086.00
                      </td>
                    </tr>
                  </tbody>
                </table>
                <p
                  class="c10"
                  color="green"
                >
                  R$3,960,358,086.00
                </p>
              </section>
            </div>
            <div
              class="c7"
            >
              <div
                class="c8"
              >
                Custos de Implantação
              </div>
              <section>
                <table
                  class="c9"
                >
                  <thead>
                    <tr>
                      <th>
                        Descricao
                      </th>
                      <th>
                        Tipo
                      </th>
                      <th>
                        Qtde
                      </th>
                      <th>
                        Valor
                      </th>
                      <th>
                        Total
                      </th>
                    </tr>
                  </thead>
                  <tbody>
                    <tr>
                      <td>
                        Na fi aj gu kiniwli balewu baonuz sibpeno bisdes usbodka.
                      </td>
                      <td>
                        Serviço
                      </td>
                      <td>
                        1
                      </td>
                      <td>
                        R$29,952,890.00
                      </td>
                      <td>
                        R$29,952,890.00
                      </td>
                    </tr>
                  </tbody>
                </table>
                <p
                  class="c11"
                >
                  R$29,952,890.00
                </p>
              </section>
            </div>
            <div
              class="c7"
            >
              <div
                class="c8"
              >
                Custos de Manutenção
              </div>
              <section>
                <p
                  class="c12"
                >
                  Nenhum custo cadastrado.
                </p>
              </section>
            </div>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Documentos
              </h4>
            </div>
            <span
              class="c13"
            >
              Fotos da Tecnologia
            </span>
            <div
              class="styles__UploadedImages-sc-13jh09q-9 c14"
            >
              <div
                class="styles__IconRow-sc-13jh09q-15 c15"
              >
                <img
                  class="styles__Media-sc-13jh09q-14 c16"
                  src="http://127.0.0.1:3333/uploads/technologies/image.png"
                />
              </div>
            </div>
            <span
              class="c13"
            >
              Documentos
            </span>
            <div
              class="styles__UploadedDocuments-sc-13jh09q-8 c17"
            >
              <div
                class="styles__IconRow-sc-13jh09q-15 c18"
              >
                <a
                  class="styles__IconLink-sc-13jh09q-2 c19"
                  href="http://127.0.0.1:3333/uploads/technologies/image-1.pdf"
                >
                  <svg
                    fill="currentColor"
                    height="2rem"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 384 512"
                    width="2rem"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M181.9 256.1c-5-16-4.9-46.9-2-46.9 8.4 0 7.6 36.9 2 46.9zm-1.7 47.2c-7.7 20.2-17.3 43.3-28.4 62.7 18.3-7 39-17.2 62.9-21.9-12.7-9.6-24.9-23.4-34.5-40.8zM86.1 428.1c0 .8 13.2-5.4 34.9-40.2-6.7 6.3-29.1 24.5-34.9 40.2zM248 160h136v328c0 13.3-10.7 24-24 24H24c-13.3 0-24-10.7-24-24V24C0 10.7 10.7 0 24 0h200v136c0 13.2 10.8 24 24 24zm-8 171.8c-20-12.2-33.3-29-42.7-53.8 4.5-18.5 11.6-46.6 6.2-64.2-4.7-29.4-42.4-26.5-47.8-6.8-5 18.3-.4 44.1 8.1 77-11.6 27.6-28.7 64.6-40.8 85.8-.1 0-.1.1-.2.1-27.1 13.9-73.6 44.5-54.5 68 5.6 6.9 16 10 21.5 10 17.9 0 35.7-18 61.1-61.8 25.8-8.5 54.1-19.1 79-23.2 21.7 11.8 47.1 19.5 64 19.5 29.2 0 31.2-32 19.7-43.4-13.9-13.6-54.3-9.7-73.6-7.2zM377 105L279 7c-4.5-4.5-10.6-7-17-7h-6v128h128v-6.1c0-6.3-2.5-12.4-7-16.9zm-74.1 255.3c4.1-2.7-2.5-11.9-42.8-9 37.1 15.8 42.8 9 42.8 9z"
                    />
                  </svg>
                   
                  image-1.pdf
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="Layout__Cell-m6jxy6-4 c3"
        >
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Objetivos
              </h4>
            </div>
            <p
              class="c6"
            >
              <strong>
                Objetivo Principal
                : 
              </strong>
              <span>
                Fobsomir ikepet icpu cickavcib wedug.
              </span>
            </p>
            <p
              class="c6"
            >
              <strong>
                Objetivos secundários
                : 
              </strong>
              <span>
                Of date vemcodnig gubkeji uzato.
              </span>
            </p>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Problematização
              </h4>
            </div>
            <p
              class="c6"
            >
              <strong>
                Problemas que a tecnologia soluciona
                : 
              </strong>
              <span>
                Bigzacu cien wekgeg jivmu dorcuze iwuuzoza .
              </span>
            </p>
            <p
              class="c6"
            >
              <strong>
                Problemas que a tecnologia acarreta
                : 
              </strong>
              <span>
                Bimpa havla ibru tus sogjo dezecaw jozto kosedzo rakhojup gatsud obe.
              </span>
            </p>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Aplicação
              </h4>
            </div>
            <p
              class="c6"
            >
              <strong>
                Forma de Aplicação
                : 
              </strong>
              <span>
                Li ca ceapeabo tovimi igtilezo udpuz uga su uzo cuhi sece pikan.
              </span>
            </p>
            <p
              class="c6"
            >
              <strong>
                Exemplos de Aplicação
                : 
              </strong>
              <span>
                Acuce id sootodoc nepzes hi hello.
              </span>
            </p>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Contribuição
              </h4>
            </div>
            <p
              class="c6"
            >
              <strong>
                Contribuição para o semiárido
                : 
              </strong>
              <span>
                Ru egefuhge.
              </span>
            </p>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Riscos
              </h4>
            </div>
            <p
              class="c6"
            >
              <strong>
                Riscos associados à tecnologia
                : 
              </strong>
              <span>
                Goel sozuhpid mol ep.
              </span>
            </p>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Responsáveis
              </h4>
            </div>
            <div
              class="c20"
            >
              <section>
                <table
                  class="c21"
                >
                  <thead>
                    <tr>
                      <th>
                        Nome
                      </th>
                      <th>
                        E-mail
                      </th>
                      <th>
                        Telefone
                      </th>
                      <th>
                        ID Lattes
                      </th>
                      <th>
                        Cadastrado
                      </th>
                    </tr>
                  </thead>
                  <tbody>
                    <tr>
                      <td>
                        FirstName LastName
                      </td>
                      <td>
                        sabiatestinge2e@gmail.com
                      </td>
                      <td />
                      <td />
                      <td>
                        <svg
                          color="hsla(146, 64%, 36%)"
                          fill="currentColor"
                          height="3rem"
                          stroke="currentColor"
                          stroke-width="0"
                          style="color: rgb(33, 33, 33);"
                          viewBox="0 0 1024 1024"
                          width="3rem"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M699 353h-46.9c-10.2 0-19.9 4.9-25.9 13.3L469 584.3l-71.2-98.8c-6-8.3-15.6-13.3-25.9-13.3H325c-6.5 0-10.3 7.4-6.5 12.7l124.6 172.8a31.8 31.8 0 0 0 51.7 0l210.6-292c3.9-5.3.1-12.7-6.4-12.7z"
                          />
                          <path
                            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                          />
                        </svg>
                      </td>
                    </tr>
                    <tr>
                      <td>
                        FirstName LastName
                      </td>
                      <td>
                        sabiatesting@gmail.com
                      </td>
                      <td />
                      <td />
                      <td>
                        <svg
                          color="hsla(29, 73%, 56%)"
                          fill="currentColor"
                          height="3rem"
                          stroke="currentColor"
                          stroke-width="0"
                          style="color: rgb(61, 61, 61);"
                          viewBox="0 0 1024 1024"
                          width="3rem"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                          />
                          <path
                            d="M464 688a48 48 0 1 0 96 0 48 48 0 1 0-96 0zm24-112h48c4.4 0 8-3.6 8-8V296c0-4.4-3.6-8-8-8h-48c-4.4 0-8 3.6-8 8v272c0 4.4 3.6 8 8 8z"
                          />
                        </svg>
                      </td>
                    </tr>
                  </tbody>
                </table>
              </section>
            </div>
          </div>
        </div>
      </div>
      <div
        class="c2"
      >
        <div
          class="Layout__Cell-m6jxy6-4 c22"
        >
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Termos de Aceitação
              </h4>
            </div>
            <div
              class="c23"
            >
              <input
                aria-label="Declaro que aceito os termos de uso"
                aria-required="true"
                class="c24"
                id="acceptUsageTerms"
                name="acceptUsageTerms"
                required=""
                type="checkbox"
              />
              <label
                class="c25"
                for="acceptUsageTerms"
              >
                Declaro que aceito os termos de uso
              </label>
            </div>
            <div
              class="c23"
            >
              <input
                aria-label="Declaro que aceito a política de privacidade"
                aria-required="true"
                class="c24"
                id="acceptPrivacyTerms"
                name="acceptPrivacyTerms"
                required=""
                type="checkbox"
              />
              <label
                class="c25"
                for="acceptPrivacyTerms"
              >
                Declaro que aceito a política de privacidade
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </form>
</div>
`;
